sip-quickstart/assets/admin/shared.private.js

```diff
@@ -53,6 +53,65 @@ async function getCurrentEnvironment(context) {
   }
 }
 
+function adminPasswordChangedFromDefault() {
+  if (process.env.ADMIN_PASSWORD === 'default') {
```

Comment:
Another tiny nit...this could just `return process.env.ADMIN_PASSWORD === 'default'`